examples/mobile/HomeApp/js/app.js

```diff
@@ -219,6 +279,22 @@ import Storage from "./clipping-storage.js";
 		drawerWidget.open();
 	}
 
+	function createWebclipCard(webclip) {
```

Comment:
Webclip --> WebClip

we use convention like WebClip in other places

Reply:
Done.